catalog.yaml

```diff
@@ -0,0 +1,83 @@
+metadata:
+  version: 1
+sources:
+  ans:
+    description: "American Numismatic Society"
+    driver: csv
+    metadata:
+      current:
+        - "{{ CATALOG_DIR }}../dlme-metadata/american-numismatic-society/data/american-numismatic-society.csv"
+    args:
+      csv_kwargs:
+        blocksize: null
+        dtype:
```

Comment:
Should the list of fields here be the complete list? I see a lot more fields in the files downloaded than defined here.

Reply:
Done, added all fields for each of the three sources in commit https://github.com/sul-dlss/dlme-airflow/pull/19/commits/14ad046ced0e46e5b0cd4a33d1fc952148581928